src/images/plots/breaks_date.R

```diff
@@ -0,0 +1,24 @@
+box::use(lubridate)
+box::use(scales)
+
+#' Generate breaks for date axis
+#'
+#' From the date axis, checks the number of years covered. If 3 or more years
+#' are included in the data, then the breaks are set to be the years. Otherwise,
+#' `scales::breaks_pretty()` is used. We check the max compared to min because
+#' it is not always the raw vector passed in, but sometimes the limits of the
+#' function.
+#'
+#' @param x Vector to generate breaks for
+#'
+#' @export
+breaks_date <- function(x) {
+  max_year <- lubridate$year(max(x))
+  min_year <- lubridate$year(min(x))
+  n_years <- max_year - min_year
+  if (n_years >= 2) {
+    scales$breaks_width(width = "1 year")(x)
+  } else {
+    scales$breaks_pretty()(x)
+  }
```

Comment:
1. in Roxy you say if 3 or more years, but in code its `n_years > = 2`.
2. I was actually thinking the logic would be opposite ... i.e if there were 10 years we'd want breaks_pretty() to do it's thing so it didn't get busy.
3. What does breaks_pretty() do if 1 year or less? it just cuts the months nicely?

Reply:
Yeah, it's actually no longer `n_years`, it's really calculating `years_diff`. I think overall can change this to be using the year breaks if 3 or more years covered up to something like 10 years. But your suggestion for the plot in Slack for instance covers 8 years, so I think we still want yearly coverage up to then or so.

`breaks_pretty()` chops up months or quarters depending on the data.